proto/swap_resolver.proto

```diff
@@ -0,0 +1,106 @@
+
+syntax = "proto3";
+
+//import "google/api/annotations.proto";
+
+//option java_multiple_files = true;
+//option java_package = "io.grpc.examples.routeguide";
+//option java_outer_classname = "RouteGuideProto";
+
+package swapresolver;
+
+// Interface exported by the server.
+service SwapResolver {
+  // ResolveHash is used by LND to request translation of Rhash to a pre-image.
+  // the resolver may return the preimage and error indicating that there is no
+  // such hash/deal
+  rpc ResolveHash(ResolveReq) returns (ResolveResp) {}
+
+}
+
+enum CoinType {
+    BTC = 0;
+    LTC = 1;
+}
+
+// 6
+message ResolveReq {
+  string hash = 1 ;
+}
+
+message ResolveResp {
+  string preimage = 1;
+}
+
+// Todo: this should be in a different proto file to keep the concep of resolver clean
```

Comment:
"concept" instead of "concep". And why not put it in a different proto file now? Seems like it should be easy to do now.

Reply:
Removed